robot_smach_states/src/robot_smach_states/.gitignore

```diff
@@ -1 +0,0 @@
-*.pyc
```

Comment:
We dont want to commit .pyc files

Reply:
Jep but is covered by the gitignore in the main folder of the repo